frontend/integration-tests/tests/modal-annotations.scenario.ts

```diff
@@ -114,6 +114,7 @@ describe('Modal Annotations', () => {
     isPresent: boolean
   ) {
     await crudAnnotationFromDetail(action, annotationKey, annotationValue);
+    await browser.get(`${appHost}/k8s/ns/${testName}/deployments/${WORKLOAD_NAME}`);
```

Comment:
It feels like we should at least wait for the loading indicator to go away after changing the URL.

I'm a little unclear why setting the URL is needed. If this helps with test reliability, great. Just want to understand :)

Reply:
It was taking some undetermined amount of time for the '# Annotations' link to update after creating or deleting an annotation.  Setting the URL instantly updates the correct number in the annotations link.